README.md

```diff
@@ -1,88 +1,333 @@
-# Mindee API helper library for Python
+# Mindee API Helper Library for Python
 
-The full documentation is available [here](https://developers.mindee.com/docs/getting-started)
+## Quick Start
+Here's the TL;DR of getting started.
 
-## Requirements
+First, get an [API Key](https://developers.mindee.com/docs/make-your-first-request#create-an-api-key)
 
-This library is officially supported on Python 3.7 to 3.10.
+Then, install this library:
+```shell script
+pip install mindee
+```
+
+Finally, Python away!
+```python
+from mindee import Client
+
+# Init a new client and pass it the Invoice API key
+mindee_client = Client().config_invoice("my-invoice-api-key")
```

Comment:
Would it be prettier as:

```python
from mindee import Client

# Init a new client and pass it the Invoice API key
client = Client()
client.add_invoice("my-invoice-api-key")
```

Reply:
Confusing, `add_invoice` makes me think of the action of adding an invoice to the client to parse it. Like a path or a file object. We are talking about the config here, the `config` keyword is mandatory to make things easy to understand by users.